code/game/area/areas.dm

```diff
@@ -201,32 +201,34 @@ GLOBAL_LIST_EMPTY(teleportlocs)
   * Sends to all ai players, alert consoles, drones and alarm monitor programs in the world
   */
 /area/proc/poweralert(state, obj/source)
+	if (area_flags & NO_ALERTS)
+		return
 	if (state != poweralm)
 		poweralm = state
 		if(istype(source))	//Only report power alarms on the z-level where the source is located.
 			for (var/item in GLOB.silicon_mobs)
 				var/mob/living/silicon/aiPlayer = item
-				if (state == 1)
```

Comment:
changes these to !state instead

Reply:
Shouldn't it be `if (state)`?

Reply:
oh yeah my bad i was looking at the ones below, basically just dont do the state == stuff

Reply:
I think I fixed them all now.